CONTRIBUTING.md

```diff
@@ -1,5 +1,50 @@
 # Contributing to Presto
 
+Thanks for your interest in Presto.  Our goal is to build a fast, scalable and reliable distributed SQL query engine for running interactive analytic queries against data sources of all sizes ranging from gigabytes to petabytes.
+
+## Getting Started
+
+Presto's [open issues are here](https://github.com/prestodb/presto/issues). Issues that would make a good first pull request for new contributors with the `beginner-task` tag. An easy way to get started helping the project is to *file an issue*. Issues can include bugs, new features, or documentation that looks outdated. For community support, [ask for help in Slack](https://join.slack.com/t/prestodb/shared_invite/enQtNTQ3NjU2MTYyNDA2LTYyOTg3MzUyMWE1YTI3Njc5YjgxZjNiYTgxODAzYjI5YWMwYWE0MTZjYWFhNGMwNjczYjI3N2JhM2ExMGJlMWM).
+
+## Contributions
+
+Presto welcomes contributions from everyone.
+
+Contributions to Presto should be made in the form of GitHub pull requests. Each pull request will
+be reviewed by a contributor or [committer](https://github.com/prestodb/presto/wiki/committers) in the project.
+Large contributions should have an associated Github issue.
+
```

Comment:
```suggestion

Code review is also form of contribution and it is actually one of the most valuable. All code has to be reviewed in order to get merged and the more reviewers we have the better the review experience for everyone.
```

Reply:
I think that doesn't go far enough to set expectations. I think we generally don't do a good job at this as a project and the result is that committers do all the review. Throughout the lifecycle of someones escalating participation in the project I think it needs to be clear that they should be reviewing as much code as they are contributing.

It also clues people into why their review might not be done immediately.

This really isn't something I feel strongly about in this specific instance, but in general I think we are struggling with this as a project.

Reply:
That's a good point.  How about this:

```
## Contributions

Presto welcomes contributions from everyone.

Contributions to Presto should be made in the form of GitHub pull request submissions and reviews. 

Each pull request submission will be reviewed by a contributor or [committer](https://github.com/prestodb/presto/wiki/committers) 
in the project. Only committers may merge a pull request. Large contributions should have an associated Github issue.

Pull request reviews are encouraged for anyone in the community who would like to contribute to Presto, and are
expected from contributors and committers in at least equal proportion to their code contributions.
```